src/components/dialogs/parameters/sensi/sensitivity-analysis-parameters.js

```diff
@@ -430,12 +442,45 @@ export const SensitivityAnalysisParameters = ({
                     <Button onClick={clear}>
                         <FormattedMessage id="resetToDefault" />
                     </Button>
-                    <SubmitButton onClick={handleSubmit(onSubmit)}>
-                        <FormattedMessage id="validate" />
-                    </SubmitButton>
-                    <CloseButton hideParameters={hideParameters} />
+
+                    <Button variant="outlined">
+                        <SubmitButton onClick={handleSubmit(onSubmit)}>
+                            <FormattedMessage id="validate" />
+                        </SubmitButton>
+                    </Button>
+                    <CloseButtonWithConfirm
+                        callback={() => {
+                            if (
+                                formState.dirtyFields &&
+                                Object.keys(formState.dirtyFields).length === 0
+                            ) {
+                                hideParameters();
+                            } else {
+                                setPopupConfirm(true);
+                            }
+                        }}
+                        label={'Annuler'}
+                    />
                 </DialogActions>
-            </Grid>
-        </FormProvider>
+            </FormProvider>
+
+            <SelectOptionsDialog
+                open={popupConfirm}
+                onClose={handleClosePopupConfirm}
+                onClick={handlePopupConfirm}
+                child={
+                    <DialogContentText>
+                        <FormattedMessage
+                            id={'Are you sur that you want to abord ?'}
+                        />
+                    </DialogContentText>
+                }
+                style={{
+                    '& .MuiPaper-root': {
+                        overflowY: 'visible',
+                    },
+                }}
```

Comment:
you don't applied it to element style directly?

Reply:
done